ruby/ql/lib/codeql/ruby/frameworks/Twirp.qll

```diff
@@ -18,8 +18,7 @@ module Twirp {
    */
   class ServiceInstantiation extends DataFlow::CallNode {
     ServiceInstantiation() {
-      this =
-        API::getTopLevelMember("Twirp").getMember("Service").getASubclass().getAnInstantiation()
+      this = API::getTopLevelMember("Twirp").getMember("Service").getAnInstantiation()
```

Comment:
Here `getASubclass` has been removed, because asking for `getAnInstantiation` will include instantiations of sub classes?

Reply:
Yes, again, from the PR description:
> The PR also removes some uses of getASubclass() which were redundant because it came before a member predicate that itself performs getASubclass(). The optimizer did not handle that very well.